tests/plugins/test_terminator.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Test Terminator
+---------------
+"""
+
+from jinja2 import Template
+
+import folium
+from folium import plugins
+
+def test_terminator():
+    m = folium.Map([45., 3.], zoom_start=1)
```

Comment:
Virtually all tests start by importing, `jinja2.Template`, `folium`, and `folium.plugins` and creating a map.  What do you think of an external `util.py` with this a simple function to do that?